Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile-upstream:master
 
 ARG RUNC_VERSION=v1.2.1
-ARG CONTAINERD_VERSION=v1.7.23
+ARG CONTAINERD_VERSION=v2.0.0
 # CONTAINERD_ALT_VERSION_16 defines fallback containerd version for integration tests
```

Comment:
(I assume not worth keeping ALT_VERSION_17)

Reply:
1.7 should become the alternative version

Reply:
No need to keep v1.6 LTS?

Reply:
I don't think so